### PR TITLE
Use "127.0.0.1" instead of "localhost"

### PR DESCRIPTION
**Pull request by malor**
Apparently, in Rocket >= 0.5 `ROCKET_ADDRESS` only accepts IP addresses but not hostnames: "Use only IP addresses for the address configuration parameter." [1]. xsnippet-api fails to start, unless we pass "127.0.0.1" (or "::1") instead of "localhost".

[1] https://rocket.rs/v0.5-rc/guide/upgrading/#configuration

### CODE REVIEW

File: roles/postgres/tasks/main.yml [hunk 1]
```diff
@@ -61,12 +61,14 @@
   ansible.builtin.template:
     src: postgres-backups@.service.j2
     dest: /usr/lib/systemd/system/postgres-backups@.service
+    mode: '644'
```
Comment (ikalnytskyi) [inline roles/postgres/tasks/main.yml]: nit: I find `mode: u=rw,g=r,o=r` is a bit more readable.

File: roles/caddy/handlers/main.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 
-- name: caddy-restart
+- name: Caddy restart
```
Comment (ikalnytskyi) [inline roles/caddy/handlers/main.yml:3]: Is this change triggered by the linter? I wonder, why it doesn't complain in my other repository? :thinking:

File: roles/volume/tasks/main.yml [hunk 3]
```diff
@@ -12,6 +12,7 @@
 - name: If not, create a new filesystem
   ansible.builtin.command: "mkfs.{{ volume_fs }} {{ volume_device }}"
   when: not volume_has_fs
+  changed_when: true
```
Comment (ikalnytskyi) [inline roles/volume/tasks/main.yml:15]: Q: Isn't it default?

**Review by ikalnytskyi — APPROVED** [review]
LGTM. I'd just ask to use `u=rw,g=r,o=r` instead of `644` for readability, and in order to align this usage with other parts of the playbook.

File: roles/caddy/handlers/main.yml (hunk 2 above)
Comment (malor) [reply roles/caddy/handlers/main.yml:3]: Yeah, it's weird :( That's `name[casing]`  from https://ansible.readthedocs.io/projects/lint/rules/name/

File: roles/volume/tasks/main.yml (hunk 3 above)
Comment (malor) [reply roles/volume/tasks/main.yml:15]: Apparently not. Or rather, the linter was complaining about not having it.

File: roles/postgres/tasks/main.yml (hunk 1 above)
Comment (malor) [reply roles/postgres/tasks/main.yml]: Done!

**Pull request merged**